rita/src/rita_client/dashboard/mod.rs

```diff
@@ -248,11 +248,11 @@ impl Handler<GetNodeInfo> for Dashboard {
                             if maybe_route.is_err() {
                                 output.push(NodeInfo {
                                     nickname: serde_json::to_string(&identity.mesh_ip).unwrap(),
-                                    route_metric_to_exit: 0,
+                                    route_metric_to_exit: 65535,
                                     total_payments: debt_info.total_payment_recieved.into(),
                                     debt: debt_info.debt.clone().into(),
-                                    link_cost: 0,
-                                    price_to_exit: 0,
+                                    link_cost: 65535,
```

Comment:
To avoid potential typos you can use `u16::max_value()`, `u32::max_value()` etc.

Reply:
There is also version with type inference: `link_cost: num::Bounded::max_value()` etc